.github/workflows/tab-finder.yml

```diff
@@ -0,0 +1,39 @@
+# Find tabs in modified text files
+name: Tab finder
+on: [push, pull_request]
+env:
+  MAIN_BRANCH: dev
+jobs:
+  build:
+    name: Tab finder
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # needed to get the full history and pull from upstream
+      - name: Pull from upstream
+        run: |
+          # Update from upstream to avoid including missing commits in the diff
+          git config --global user.name "Nemo" # required on some servers
+          git config pull.rebase false
+          git remote add upstream https://github.com/AliceO2Group/AliceO2.git || exit 1
```

Comment:
Is this necessary? Don't we get the remote from the clone step above already?

Reply:
Doesn't the checkout action just clone the fork remote? Is the main branch in the fork remote is not up to date, diff against upstream would include unrelated files.